CHANGELOG.md

```diff
@@ -4,6 +4,20 @@
 
 [1]: https://pypi.org/project/google-cloud-pubsub/#history
 
+## [1.2.0](https://www.github.com/googleapis/python-pubsub/compare/v1.1.0...v1.2.0) (2020-02-05)
+
+
+### Features
+
+* **pubsub:** add delivery attempt property to message object received by user code ([#10205](https://www.github.com/googleapis/python-pubsub/issues/10205)) ([a0937c1](https://www.github.com/googleapis/python-pubsub/commit/a0937c13107b92271913de579b60f24b2aaac177))
```

Comment:
This and the other issue links are dead, it seems that the links should still point the the old repo where those (closed) issues exist?

It's probably just a transitional problem, since a release was made in the new repo, but the fixes it contains are still from the old monorepo.

Reply:
The old repo was deleted so we can't fix the links. At best, we can remove them to prevent confusion.

It looks like a transitional problem, yes.